src/libraries/Microsoft.Extensions.Hosting/src/HostOptions.cs

```diff
@@ -30,11 +39,25 @@ public class HostOptions
         internal void Initialize(IConfiguration configuration)
         {
             var timeoutSeconds = configuration["shutdownTimeoutSeconds"];
-            if (!string.IsNullOrEmpty(timeoutSeconds)
+            if (!string.IsNullOrWhiteSpace(timeoutSeconds)
                 && int.TryParse(timeoutSeconds, NumberStyles.None, CultureInfo.InvariantCulture, out var seconds))
             {
                 ShutdownTimeout = TimeSpan.FromSeconds(seconds);
             }
+
+            var backgroundServiceStopBehavior = configuration["backgroundServiceStopBehavior"];
+            if (!string.IsNullOrWhiteSpace(backgroundServiceStopBehavior)
+                && Enum.TryParse<BackgroundServiceStopBehavior>(backgroundServiceStopBehavior, out var stopBehavior))
+            {
+                BackgroundServiceStopBehavior = stopBehavior;
+            }
+
+            var backgroundServiceExceptionBehavior = configuration["backgroundServiceExceptionBehavior"];
```

Comment:
So the reason I added these was primarily for tests - there was already a method in place to have configuration values set and for those to be honored in the test when setting up an IHost.. and i thought.. why aren't these supported?